src/QirRuntime/README.md

```diff
@@ -55,35 +49,53 @@ Running cmake from the editors will likely default to MSVC or clang-cl and fail.
 
 See [https://code.visualstudio.com/docs/remote/wsl] on how to use VS Code with WSL.
 
-## Test
+#### Other Prerequisites
+The build depends on `Microsoft.Quantum.Simulator.Runtime` dynamic library built at a higher level of the directory tree.
+To build that library follow the instructions in [`qsharp-runtime/README.md`](../../README.md#building-from-source)
+(up to and including the step `Simulation.sln`).
 
-Some of the tests depend on Microsoft.Quantum.Simulator.Runtime library. To run them make sure to build Native simulator
- from this repository or provide your own version of the library in a folder the OS would search during dynamic library
- lookup.
 
-Some of the tests use generated QIR (*.ll) files as build input. Currently the files are checked-in as part of the project
- but in the future they will be replaced by automatic generation during build. To regenerate the files, run generateqir.py
- or build/test scripts without specifying `noqirgen`. To use the checked-in files without regenerating them, run build/test
- scripts with `noqirgen` argument.
+### Build Commands
+To build QirRuntime you can run [`build-qir-runtime.ps1`](build-qir-runtime.ps1) script from QirRuntime folder:
+```batch
+pwsh build-qir-runtime.ps1
+```
+  
+The script will create the `build/{Debug|Release}` folder and place the build artifacts in it. The configuration `Debug|Release`
+is specified with the `BUILD_CONFIGURATION` environment variable.
+If the variable is not set then the default is specified in [`set-env.ps1`](../../build/set-env.ps1).  
+We strongly recommend doing local builds using this [`build-qir-runtime.ps1`](build-qir-runtime.ps1) build script 
+because it is used by the continuous integration (CI) infrastructure, and this script also runs clang-tidy if it is installed.
 
-### Running tests with test.py
+Or you can use CMake directly (not recommended). For example in Windows you can enter the following commands in the command line:
+```batch
+@rem Navigate to QirRuntime folder.
```

Comment:
This looks like it uses cmd notation, but cmd is no longer the default shell on Windows (PowerShell was made the default several years ago).

Reply:
Yes... I'm out-of-date ;-)
Don't you have on top of your head, what I should use instead of  ```batch
for PowerShell syntax highlighting? Locally I try `ps`, `ps1`, `PowerShell` but my MarkDown plugin for Chrome does not highlight the syntax if I use any of those.

Reply:
Oh. looks like it is shown [here](https://github.com/MicrosoftDocs/PowerShell-Docs/issues/1511).